daemon/k8s_watcher_test.go

```diff
@@ -201,12 +201,12 @@ func (ds *DaemonSuite) Test_missingCNPv2(c *C) {
 			name: "repository contains all policies",
 			setupArgs: func() args {
 				p1 := policy.NewPolicyRepository()
-				_, err := p1.Add(api.Rule{
+				_, _, err := p1.Add(api.Rule{
 					EndpointSelector: api.NewESFromLabels(labels.ParseSelectLabel("id=a")),
 					Labels: utils.GetPolicyLabels(
 						"foo", "bar", uuid,
 						utils.ResourceTypeCiliumNetworkPolicy),
-				})
+				}, map[uint16]*identity.Identity{})
```

Comment:
Make this a type and and add a utility function to construct a new instance of it.